consensus-types/blocks/execution.go

```diff
@@ -197,6 +202,16 @@ func (executionPayload) ValueInGwei() (uint64, error) {
 	return 0, consensus_types.ErrUnsupportedField
 }
 
+// DepositReceipts --
+func (e executionPayload) DepositReceipts() ([]*enginev1.DepositReceipt, error) {
+	return nil, consensus_types.ErrUnsupportedField
```

Comment:
Is this a TODO for me to resolve when working on 6110?

Reply:
In this struct, no. This is just to comply with the interface changes. executionPayloadHeaderElectra is able to return the deposit receipts